components/BottomNavBar.js

```diff
@@ -0,0 +1,42 @@
+import { View, Text, StyleSheet } from 'react-native';
+
+
+const BottomNavBar = () => {
+
+    return (
+        <View style={styles.bottomNavBar}>
+            <Text style={styles.title}> 🏠      📅      🪷      📄 </Text>
```

Comment:
I really like the little emojis! Perfect addition to the navBar... curious on how we can connect these to event listeners. Maybe need to reconfigure when we get to the onClicks?